wgengine/magicsock/magicsock.go

```diff
@@ -3670,6 +3670,17 @@ func betterAddr(a, b addrLatency) bool {
 	if a.IsZero() {
 		return false
 	}
+	if a.IP.Is6() && b.IP.Is4() {
+		// Prefer IPv6 for being a bit more robust, as long as
+		// the latencies are roughly equivalent.
+		if a.latency/10*9 < b.latency {
```

Comment:
I don't understand this comparison. Why are you dividing one side and not the other? How does that translate to "roughly equivalent" ?

Reply:
They're in the same ballpark latency-wise from our single data point. It's not 10% worse.

Reply:
There are test cases with comments too.

Reply:
Oh, I misread that as /(10^9), and parsed it as "nanoseconds to seconds conversion", at which point the compare makes no sense.

Okay, yeah, now I'm reading it right, LGTM.